website/content/docs/concepts/domain-model/targets.mdx

```diff
@@ -36,6 +36,9 @@ A target has the following configurable attributes:
 
 - `description` - (optional)
 
+- `type` - (required)
+  Defines the type of connection to a target. Options are `static`, `tcp`, and `ssh`. Even if the type of connection to the target is `ssh`, you should select the `tcp` subcommand. Using the `ssh` command is reserved for [credential injection](https://developer.hashicorp.com/boundary/tutorials/hcp-administration/hcp-ssh-cred-injection)
```

Comment:
I believe the only valid target types are `tcp` and `ssh`, not `static`.

Also looking at the other domain model docs pages, we don't tend to include a `type` attribute in the list here. Instead the subtypes are listed below under sub-headings. Perhaps that would be a better place to call out that SSH Targets are required for credential injection, that you can only use SSH Targets if you want to use credential injection?

Reply:
Hey @tmessi , thanks for the feedback. Apologies on the 'static' reference. I could have sworn that I read that somewhere but probably got confused. I agree, perhaps under the SSH target attributes HCP/ENT section we could change the wording to:

`"SSH targets can source username/password or SSH private key credentials from Vault [credential libraries](https://developer.hashicorp.com/boundary/docs/concepts/domain-model/credential-libraries) or static [credentials](https://developer.hashicorp.com/boundary/docs/concepts/domain-model/credentials). If using injected crednetials, Boundary then injects the credentials into the SSH session between a client and end host. This allows users to securely connect to remote hosts using SSH, while never being in possession of a valid credential for that target host.

If the target type is set to SSH then brokered or injected credentials has to be used. You cannot reference a static .pem file to use`

Or something like that @Dan-Heath

Reply:
It might help to state clearly the requirements around SSH Targets and injected application credentials:

- An SSH Target **must** have at least one injected application credential.
- A TCP Target **can not** have any injected application credentials.

Therefore:
- If you want to use brokered credentials when connecting to a target running SSH, you must use a TCP Target.
- If you want to use injected application credentials when connecting to a target running SSH, you must use an SSH Target.
- Also if you want to enable session recording for a target running SSH, you must use injected application credentials and an SSH Target.